doc/release/1.15.0-notes.rst

```diff
@@ -15,6 +15,14 @@ New functions
 * `np.ma.stack`, the `np.stack` array-joining function generalized to masked
   arrays.
 
+* `np.printoptions`, the context manager which sets print options temporarily
+  for the scope of the ``with`` block
+
+```
+    >>> with np.printoptions(precision=2):
+    ...     print(np.array([2.0])) / 3
+    [0.67] 
+``` 
```

Comment:
I think backticks are markdown-only. Two colons after the word "block" might do the right thing.

Reply:
That's correct.

Reply:
Indeed, fixed and force-pushed.